python/lsst/ts/ataos/ataos_csc.py

```diff
@@ -1197,6 +1221,115 @@ async def set_hexapod(self, azimuth, elevation, axis=f'xyzuvw'):
                 # correction completed... flip bit on detailedState
                 self.detailed_state = self.detailed_state ^ status_bit
 
+    async def set_atspectrograph_corrections(self, azimuth, elevation):
```

Comment:
If you let the correction loop apply the hexapod offset you won't need az/el.

Reply:
see comment in conversation about this suggestion/question